### DISCUSSION TITLE
Stat panel: Set thousands separator format with cutstom unit

### DISCUSSION THREAD

**Discussion by baur** [grafana/grafana#38247] [Dashboards, Panels & Visualizations]
**What would you like to be added**:

Set thousands separator format with cutstom unit

**Why is this needed**:

Unreadable numbers with custom unit

![image](https://user-images.githubusercontent.com/2898361/112802587-2b3e6100-9094-11eb-91e2-94a422a2910f.png)

I think format and unit must be as separate options

**Comment by Elfo404** [comment]
(partially) related to https://github.com/grafana/grafana/issues/6193 and https://github.com/grafana/grafana/issues/4252

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
